cmd/exec.go

```diff
@@ -42,27 +42,27 @@ func execRun(cmd *cobra.Command, args []string) error {
 		return errors.New("must specify command to run")
 	}
 
-	service := strings.ToLower(args[0])
-	if err := validateService(service); err != nil {
-		return err
-	}
-
 	command := commandPart[0]
 
 	var commandArgs []string
 	if len(commandPart) > 1 {
 		commandArgs = commandPart[1:]
 	}
 
+	env := environ(os.Environ())
 	secretStore := store.NewSSMStore()
-	secrets, err := secretStore.List(service, true)
-	if err != nil {
-		return err
-	}
+	for _, service := range args {
+		if err := validateService(service); err != nil {
```

Comment:
The logic for validateService changed when you moved the toLower below this function.